Makefile

```diff
@@ -84,7 +84,7 @@ only: sub-only this-only
 test-suite: sub-test-suite this-test-suite
 
 clean: sub-clean this-clean
-	rm -f coq.hb
+	# rm -f coq.hb
```

Comment:
@gares 
With Coq 8.18 the compilation fails with this error
```
ocamlc unix.cma str.cma -g hb.ml -o coq.hb
File "hb.ml", line 1:
Error: Error while linking /nix/store/bink43irwl4lp7fli0lh6j82zsrkycp5-ocaml-4.14.1/lib/ocaml/unix.cma(Unix):
       The external function `unix_realpath' is not available
```
Hence my commenting of the `coq.hb`.
I seem to recall this used to achieve plan B... but I think we kind of dropped plan B now, so should we remove this executable?

Reply:
yes